index.html

```diff
@@ -17,13 +17,17 @@ <h1 class="logo"><a href=""><img src="images\logo.png" alt="Gardener"></a></h1>
               <li><a href="#contact">Contact</a></li>
         </ul>
       </nav>
+      <div class="nav-icon">
```

Comment:
You need to hide this block of div in desktop mode currently, this is causing layout break in the desktop view.